src/main/java/dan200/computercraft/core/logger/Logger.java

```diff
@@ -0,0 +1,65 @@
+package dan200.computercraft.core.logger;
+
+import javax.annotation.Nonnull;
+import java.util.logging.Level;
+
+public class Logger
+{
+    private static ILogger instance;
+
+    @Nonnull
+    public static ILogger getInstance()
```

Comment:
Any worries about this being called from two threads at the same time and causing two `ILogger` instances to be created?